applovin_max/example/android/app/build.gradle

```diff
@@ -1,3 +1,9 @@
+plugins {
```

Comment:
I think you might need to merge master into this branch so your changes from https://github.com/AppLovin/AppLovin-MAX-Flutter/pull/188 do not show in this PR?

Reply:
I somehow rebased the master and now had to force push to remove it.